README.md

```diff
@@ -78,6 +79,35 @@ and you are all set!
 
 ## Help & Contribute
```

Comment:
It would be better if we move this to contributing.md and leave a link in the README